common/djangoapps/student/tests/test_email.py

```diff
@@ -233,78 +236,42 @@ def assertFailedRequest(self, response_data, expected_error):
         self.assertEquals(expected_error, response_data['error'])
         self.assertFalse(self.user.email_user.called)
 
-    def test_unauthenticated(self):
-        self.request.user = AnonymousUser()
-        self.request.user.email_user = Mock()
-        with self.assertRaises(Http404):
-            change_email_request(self.request)
-        self.assertFalse(self.request.user.email_user.called)
-
-    def test_invalid_password(self):
-        self.request.POST['password'] = 'wrong'
-        self.assertFailedRequest(self.run_request(), 'Invalid password')
```

Comment:
This error was thrown in the wrapper before validate_new_email and do_email_change_request were called-- no longer needed.